### PR TITLE
Borrow/repay check removal

### PR DESCRIPTION
**Pull request by The-3D**

### CODE REVIEW

File: test-suites/validation-logic.spec.ts [hunk 1]
```diff
@@ -404,132 +404,6 @@ makeSuite('ValidationLogic: Edge cases', (testEnv: TestEnv) => {
     ).to.be.revertedWith(RESERVE_INACTIVE);
   });
 
-  it('validateRepay() when variable borrowing and repaying in same block (revert expected)', async () => {
```
Comment (miguelmtzinf) [inline test-suites/validation-logic.spec.ts:407]: I believe its better to add these tests cases as non-reverting

**Review by miguelmtzinf — CHANGES_REQUESTED** [review]

File: test-suites/validation-logic.spec.ts (hunk 1 above)
Comment (miguelmtzinf) [reply test-suites/validation-logic.spec.ts:407]: Added tests

**Review by miguelmtzinf — APPROVED** [review]

**Comment by kyzia551** [comment]
looks fine
